notary-commons/src/main/kotlin/sidechain/iroha/util/RequestHelper.kt

```diff
@@ -89,14 +89,48 @@ private fun getAccount(queryAPI: QueryAPI, accountId: String): QryResponses.Acco
     val res = queryAPI.api.query(q)
     if (res.hasErrorResponse()) {
         val errorResponse = res.errorResponse
-        throw Exception(
-            "Cannot get account. " +
-                    "Error code ${errorResponse.errorCode} ${errorResponse.reason} ${errorResponse.message}"
-        )
+        throw Exception("Cannot get account. ${getErrorMessage(errorResponse)}")
     }
     return res.accountResponse
 }
 
+/**
+ * Returns raw query response with Iroha block.
+ * May be used to handle Iroha error codes manually
+ * @param queryAPI - Iroha queries network layer
+ * @param height - height of Iroha block to get
+ * @return Iroha block
+ */
+fun getBlockRawResponse(queryAPI: QueryAPI, height: Long): QryResponses.QueryResponse {
+    val q = Query.builder(queryAPI.accountId, queryCounter.getAndIncrement())
+        .getBlock(height)
+        .buildSigned(queryAPI.keyPair);
+    return queryAPI.api.query(q);
+}
+
+/**
+ * Returns Iroha block by its height
+ * @param queryAPI - Iroha queries network layer
+ * @param height - height of Iroha block to get
+ * @return Iroha block response
+ */
+fun getBlock(queryAPI: QueryAPI, height: Long): QryResponses.BlockResponse {
```

Comment:
it is not used

Reply:
This is not used yet. I hope it will be used in the future.